docs/Technical-Documentation/error-categories-guide.md

```diff
@@ -0,0 +1,82 @@
+# Error Categories Guide
+
+## Category Definitions
+
+Parity with the legacy system includes categories 1-4
+
+1. **Pre-parsing errors** — These types of errors are considered to be violations of the expected record layout, so the records are not "eligible" to be parsed or validated. See existing reference material (in UX murals and parsing notebooks) about what these errors are and how to detect them.
+
+   Ex:
+
+   ```Your file does not begin with a header record```,
+
+   ```Header length is 2 characters but must be 23```
+
+2. **Out-of-range value errors –** These are based on the abovementioned instructions.
```

Comment:
if we are referencing the instructions, we should link to them. they are referenced in the frontend resource cards.